### PR TITLE
[Cleanup] Remove old sinks

### PR DESCRIPTION
**Pull request by paul-amonson**

### CODE REVIEW

File: docker-deploy/hw1-config/ProviderInventoryNetworkForeignBus.json [hunk 1]
```diff
@@ -6,18 +6,15 @@
   "networkStreams": {
     "stateChangeSource": {
       "arguments": {
-        "connectPort": 8080,
-        "connectAddress": "api-gw-service-nmn.local",
-        "urlPath": "/apis/smd/hsm/v1/Subscriptions/SCN",
+        "fullUrl": "http://api-gw-service-nmn.local:443/apis/smd/hsm/v1/Subscriptions/SCN?stream_id=dai-inventory",
```
Comment (yhuan47) [inline docker-deploy/hw1-config/ProviderInventoryNetworkForeignBus.json:9]: Do we change this to hms statechange telemetry?

File: docker-deploy/hw1-config/ProviderMonitoringNetworkForeignBus.json [hunk 2]
```diff
@@ -5,42 +5,75 @@
   },
 
   "networkStreams": {
-    "telemetry": {
+    "energyTelemetry": {
       "arguments": {
-        "connectPort": 8080,
-        "connectAddress": "api-gw-service-nmn.local",
-        "urlPath": "/v1/stream/telemetry",
+        "fullUrl": "https://api-gw-service-nmn.local/v1/stream/cray-telemetry-energy?stream_id=dai-telemtry",
```
Comment (yhuan47) [inline docker-deploy/hw1-config/ProviderMonitoringNetworkForeignBus.json:10]: isn't the full link https://api-gw-service-nmn.local/apis/sma-telemetry-api/v1/stream/xxxxx?

File: docker-deploy/hw1-config/ProviderMonitoringNetworkForeignBus.json [hunk 3]
```diff
@@ -5,42 +5,75 @@
...
+        "clientId": "admin-client",
+        "clientSecret": "dd0f8dee-f40b-4b16-a5dd-af2ed1b8a99c",
+        "realm": "shasta"
       },
-      "name": "sse"
+      "name": "eventSource"
+    },
+    "powerTelemetry": {
+      "arguments": {
+        "fullUrl": "https://api-gw-service-nmn.local/v1/stream/cray-telemetry-power?stream_id=dai-telemtry",
```
Comment (yhuan47) [inline docker-deploy/hw1-config/ProviderMonitoringNetworkForeignBus.json:22]: Full url need to be updated to https://api-gw-service-nmn.local/apis/sma-telemetry-api/v1/stream/xxxxx

File: docker-deploy/hw1-config/ProviderInventoryNetworkForeignBus.json (hunk 1 above)
Comment (paul-amonson) [reply docker-deploy/hw1-config/ProviderInventoryNetworkForeignBus.json:9]: I think this is a rebase issues. Lets merge #167 first then I will revisit this review.

File: docker-deploy/hw1-config/ProviderMonitoringNetworkForeignBus.json (hunk 2 above)
Comment (paul-amonson) [reply docker-deploy/hw1-config/ProviderMonitoringNetworkForeignBus.json:10]: I think this is a rebase issues. Lets merge #167 first then I will revisit this review.

File: docker-deploy/hw1-config/ProviderMonitoringNetworkForeignBus.json (hunk 3 above)
Comment (paul-amonson) [reply docker-deploy/hw1-config/ProviderMonitoringNetworkForeignBus.json:22]: I think this is a rebase issues. Lets merge #167 first then I will revisit this review.

File: docker-deploy/hw1-config/ProviderInventoryNetworkForeignBus.json (hunk 1 above)
Comment (yhuan47) [reply docker-deploy/hw1-config/ProviderInventoryNetworkForeignBus.json:9]: Sounds good!

**Comment by paul-amonson** [comment]
This had a serious rebase error. Look for a new review.

**Pull request closed without merging**